src/mmw/js/src/draw/views.js

```diff
@@ -73,6 +74,19 @@ function validateShape(polygon) {
     return d.promise();
 }
 
+function validateClickedPointWithinDRB(latlng) {
+    var point = L.marker(latlng).toGeoJSON(),
+        d = $.Deferred();
+    var boundaries = settings.get('boundary_layers');
+    var drbPerimeter = _.findWhere(boundaries, {code: 'drb_streams'}).perimeter;
```

Comment:
One of our style conventions is to use a single `var` declaration when we need to define multiple variables like this.